website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -593,7 +619,7 @@ mutation {
 }
 ```
 
-**Querying compile SQL with saved queries** 
```

Comment:
I don't think we need examples for both `createQuery` and `compileSql` for saved queries. I would vote to delete the `compileSql` example and move the `createQuery` one up to be with the other `createQuery` examples.